src/smart-components/Conversation/components/ConversationScroll.jsx

```diff
@@ -81,13 +81,15 @@ export default class ConversationScroll extends Component {
       allMessages,
       scrollToMessage,
       useReaction,
+      replyType,
       emojiAllMap,
       editDisabled,
       deleteMessage,
       updateMessage,
       resendMessage,
       renderCustomMessage,
       renderChatItem,
+      animatedMessageId,
```

Comment:
Any UI reasons for having separate `animatedMessageId` and `highLightedMessageId `

Reply:
Styling is different?

Reply:
Yes they have a different animation
MessageSearch: color change / QuoteReply: bouncing